### DISCUSSION TITLE
wails://wails URL can be blocked

### DISCUSSION THREAD

**Discussion by stevealliance** [wailsapp/wails#2392] [General]
I have several users that are testing a wails App and for some reason the app isn't loading at all but I know the process started.
I've build the App using Lorca and Golang and this version works everywhere. My only problem it's when I use wails nothing is working on a specific Geo. I know in my office we are in a block what we want, but in that office they block everything and whitelist if needed.

In production I saw this URL (wails://) and I believe this could be blocked by IT

**Comment by stffabi** [comment]
Thanks for opening this discussion.

Maybe you could give us some more details to narrow this down, like what platform are you targeting? Did you try a debug build and have a look into the web inspector console or the stdout console output? Does the window appear and is only the frontend content missing or is there no window at all?

**Reply by stevealliance** [reply]
@stffabi 

No the app doesn't start at all  and I don't think it's an OS issue or a problem with Wails. I just want to know what will be the problem if wails://wails/ URL return nothing or 404 what we should expect 
1. Wails don't start the graphical interface (Yes or No)
2. Wails display a blank window

Users in that geo, get no error and when I gave them the debug version they get a terminal window that run all the Go code.
Only the code share with the interface (IPC) isn't executed that hint me on wails://wails might be blocked at the network level.

 Thanks

**Reply by leaanthony** [reply]
That protocol shouldn't hit the network, it *should* be in memory only.

If you are getting a 404 then somehow your memory access is being blocked or you haven't embedded the files. Even then you should see a default page telling you this. And yeah the window would show.

**Reply by stevealliance** [reply]
@leaanthony thanks for the answer that's gaving me more insight on how wails behave on the background....
Again I'm trying to see what our IT dept is actually blocking. Because the app (in production) work perfectly.
We have some user in a specific office that are able to install the production version, but the App never  start when they click on the Icon (Windows) I did some simulation on my side and I have no problem at all (Win and Mac).

That's the reason I'm trying to see if something specific to Wails might be blocked or removed by our security.

Thanks
